_pages/en_US/troubleshooting.txt

```diff
@@ -155,3 +155,15 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 1. Your device is bricked
 1. You will need to get an ntrboot-comptible flashcart (one of the ones on [this list](ntrboot) or a [hardmod](https://gbatemp.net/threads/414498/), or repair / replace your device
 1. For support (in English), join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)
+
+## WiFi connection errors after installing CFW
```

Comment:
This is a bit vague. The only case we're really worried about is system updating, as Nintendo doesn't offer any error codes when something goes wrong during system updates. If an error occurs in the browser or in eShop, there is usually an error code that goes along with it. We're not dealing with browser or eShop connection issues in this portion of the guide, so I don't think it should be so vague.

This also means an online services ban could be misinterpreted as a WiFi connection error.